app/views/compute_resources_vms/form/_openstack.html.erb

```diff
@@ -12,7 +12,7 @@
 </div>
 <%= select_f f, :tenant_id, compute_resource.tenants, :id, :name, {}, :label => _('Tenant') %>
 
-<%= select_f f, :security_groups, compute_resource.security_groups, :name, :name, {}, :label => _("Security group")  %>
+<%= select_f f, :security_group, compute_resource.security_groups, :name, :name, {}, :label => _("Security group")  %>
```

Comment:
Well, after poking around for a while, I'm sure you're right and it's a total fluke this fix works. 

There's something funny going on with setting `security_groups`, and I suspect it's on the Foreman side's magic for the CR VM forms, or a combination of that + Fog.